blueflood-http/src/test/java/com/rackspacecloud/blueflood/tracker/TrackerTest.java

```diff
@@ -268,9 +269,17 @@ public void testTrackTenant() throws Exception {
         tracker.track(httpRequestMock);
 
         // verify
-        verify(loggerMock, atLeastOnce()).info("[TRACKER] tenantId " + tenantId + " added.");
-        verify(loggerMock, atLeastOnce()).info((String)argCaptor.capture());
-        assertThat(argCaptor.getValue().toString(), containsString("[TRACKER] POST request for tenantId " + tenantId + ": /v2.0/" + tenantId + "/ingest?param1=value1&param1=value2\n" +
+        verify(loggerMock, atLeastOnce()).info(argCaptor.capture());
+        List<String> allValues = argCaptor.getAllValues();
+        for (int i = 0; i < allValues.size(); i++) {
+            System.out.println("logged " + i + ": " + allValues.get(i));
+        }
+        assertThat(allValues.size(), equalTo(3));
+        // Where does "all tenants removed" come from? I don't know. I don't grok this thing yet, but this is what
```

Comment:
`setup()` has a call to `tracker.removeAllTenants()` triggering this log?

Reply:
yeah, you're right